packages/agents/src/tests/worker.ts

```diff
@@ -191,6 +191,14 @@ export class TestOAuthAgent extends Agent<Env> {
     return new Response("Test OAuth Agent");
   }
 
+  // Allow tests to configure OAuth callback behavior
+  configureOAuthForTest(config: {
+    successRedirect?: string;
+    errorRedirect?: string;
+  }): void {
+    this.mcp.configureOAuthCallback(config);
```

Comment:
hmm should this be wrapped with a process.env.NODE_ENV check? other wise feels like a security loophole.

Reply:
This code is test-only, so should be okay? Not exposed to users.

Reply:
Oh right it's inside tests, missed it